From 893786413bfa086e63e090e8b40e038b8fcbc3b5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ga=C3=ABl=20Bonithon?= <gael@xfce.org>
Date: Sat, 13 Nov 2021 17:42:54 +0100
Subject: [PATCH 1/2] build: Honor installation prefix also for the pixbuf
 loader

---
 configure | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.orig b/configure
index 18e0f83..3d3a384 100755
--- a/configure
+++ b/configure
@@ -18908,6 +18908,7 @@ printf "%s\n" "yes" >&6; }
 printf "%s\n" "#define HAVE_LIBGDKPIXBUF 1" >>confdefs.h
 
 		      GDK_PIXBUF_DIR=`$PKG_CONFIG --variable=gdk_pixbuf_moduledir gdk-pixbuf-2.0`
+		      GDK_PIXBUF_DIR=$libdir${GDK_PIXBUF_DIR#$($PKG_CONFIG --variable=libdir gdk-pixbuf-2.0)}
 									if test -z "$GDK_PIXBUF_DIR" ; then
 				GTK_VERSION=`$PKG_CONFIG --variable=gtk_binary_version gtk+-2.0`
 				GDK_PIXBUF_DIR="$libdir/gtk-2.0/$GTK_VERSION/loaders"
-- 
2.35.1

